pkg/universe.dagger.io/go/test/build.cue

```diff
@@ -17,6 +17,7 @@ dagger.#Plan & {
 		simple: {
 			build: go.#Build & {
 				source: client.filesystem."./data/hello".read.contents
+                buildvcs: false
```

Comment:
I probably should add a 2nd test entry here (one with, one without the flag), right ?